node/packages/client/src/types.ts

```diff
@@ -55,9 +55,16 @@ export enum TransferType {
   Receive = 'receive',
 }
 
+export enum TxParser {
+  Cosmos = 'cosmos',
+  Yearn = 'yearn',
+  UniV2 = 'uniV2',
+  ZRX = 'zrx',
+  THOR = 'thor',
```

Comment:
Referenced as `Thor` throughout files
```suggestion
  Thor = 'thor',
```